tests/receivers/adopted/redisreceiver/redisreceiver_test.go

```diff
@@ -0,0 +1,29 @@
+package tests
+
+import (
+	"path"
+	"testing"
+
+	"github.com/signalfx/splunk-otel-collector/tests/testutils"
+)
+
+func TestRedisReceiverProvidesAllMetrics(t *testing.T) {
```

Comment:
If the configs and expected metrics are the same w/ and w/o client activity I would remove this server-only test.

Reply:
Removed server-only testcase